### PR TITLE
sdelete, sdelete64: add page

### PR DESCRIPTION
**Pull request by HorizonChaser**
Add tldr page of sdelete.exe on Windows, together with its 64-bit version sdelete64.exe.

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/windows/sdelete.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+> Securely delete file and directory from disk, or clean the free space on a volume or physical disk.
+> More information: <https://learn.microsoft.com/en-us/sysinternals/downloads/sdelete>.
+
+- Delete files with 3 [p]asses:
+
+`sdelete -p 3 {{path\to\file1}} {{path\to\file2 ...}}`
+
+- Delete folders and its [s]ubdirectories with 1 pass (default):
+
+`sdelete -s {{path\to\directory1}} {{path\to\directory2 ...}}`
```
Comment (spageektti) [inline pages/windows/sdelete.md:12]:
```suggestion
`sdelete -s {{path\to\directory1 path\to\directory2 ...}}`
```
It should be in one placeholder, see style guide.

File: pages/windows/sdelete.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# sdelete
+
+> Securely delete file and directory from disk, or clean the free space on a volume or physical disk.
+> More information: <https://learn.microsoft.com/en-us/sysinternals/downloads/sdelete>.
+
+- Delete files with 3 [p]asses:
+
+`sdelete -p 3 {{path\to\file1}} {{path\to\file2 ...}}`
```
Comment (spageektti) [inline pages/windows/sdelete.md:8]:
```suggestion
`sdelete -p 3 {{path\to\file1 path\to\file2 ...}}`
```

File: pages/windows/sdelete.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# sdelete
+
+> Securely delete file and directory from disk, or clean the free space on a volume or physical disk.
```
Comment (spageektti) [inline pages/windows/sdelete.md:3]:
```suggestion
> Securely delete file/directory from disk, or clean the free space on a volume/physical disk.
```

**Review by spageektti — COMMENTED** [review]
Hi @HorizonChaser. Thanks for your contribution.

**Review by gutjuri — APPROVED** [review]

**Review by spageektti — APPROVED** [review]

**Pull request merged**